### DISCUSSION TITLE
Shows error creating app after adding app's name in create app section. Not able to create an app because of this error.

### DISCUSSION THREAD

**Discussion by ShivaaneeSharma** [Budibase/budibase#4969] [Help]
![Screenshot 2022-03-21 115814](https://user-images.githubusercontent.com/102014698/159214145-43c447ba-3474-40f9-b56f-de97638626cd.jpg)

**Comment by Samirmabrouk** [comment]
Suggestiv

**Reply by mjashanks** [reply]
Hmm strange.

Could you do me a favour and...
1. Option your browser's dev-tool
2. Click "Create App"
3. Goto the `console` tab
4. Copy and errors in there to here.
5. Goto the `network` tab
6. Look for any railed requests
7. Paste the requests and responses from failed ones in here

Thank you!
